### PR TITLE
TEP-0075: Add a pipeline run example with both object param and result

### PR DESCRIPTION
**Pull request by chuangw6**
# Changes

The pipeline consists of two tasks
- task 1 writes the values of an object param to an object result.
- task 2 uses task1's object result as the value of its own object param,
and echo that object param.

/kind misc

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [ ] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
Add a pipeline run example with both object param and result
```

### CODE REVIEW

**Comment by ywluogg** [comment]
Isn't https://github.com/tektoncd/pipeline/pull/5144/files a subset of these? Can they be combined?

**Review by lbernick — APPROVED** [review]

File: examples/v1beta1/pipelineruns/alpha/pipeline-object-param-and-result.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  params:
+    - name: arg
+      type: object
+      properties:
+        foo:
+          type: string
+        bar:
+          type: string
+  steps:
+    - name: write-result
```
[the remark this answers is not in the document]
Comment (chuangw6) [reply examples/v1beta1/pipelineruns/alpha/pipeline-object-param-and-result.yaml]: good catch! changed. Thanks

**Comment by chuangw6** [comment]
> Isn't https://github.com/tektoncd/pipeline/pull/5144/files a subset of these? Can they be combined?

sg! Just removed the old one since this new file covers more use cases.

**Pull request merged**